pyramid_oereb/lib/records/plr.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, theme, legende_text, law_status, published_from, published_un
         assert isinstance(geometries, list)
         assert len(geometries) > 0
 
-        self.legende_text = legende_text
+        self.legende_text = legend_entry.legend_text
```

Comment:
Do we need both or only the first one?
self.legend_entry = legend_entry
self.legende_text = legend_entry.legend_text

Reply:
I would prefer first. And in Addition a method/property on the record class like this:

```
@property
def legend_text(self):
    return self.legend_entry.legend_text
```